infra/src/app-user.ts

```diff
@@ -0,0 +1,20 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+import { nm } from "./helpers";
+import { writeExportBucket } from "./models";
+
+export const appUser = new aws.iam.User(nm("app"), {
+  name: nm("app"),
+});
+
+new aws.iam.UserPolicyAttachment(nm("app"), {
+  user: appUser.name,
+  policyArn: writeExportBucket.arn,
```

Comment:
Do you plan to create a new bucket for exporting logs, or write them to the some one we use fro exported models?

If the latter, I'd prefer a name change to reflect that the bucket no longer contains just models.

Reply:
I think using the same bucket for both is fine. We can change the name at some point but didn't seem like a priority atm.